### PR TITLE
Refactor display board

### PR DESCRIPTION
**Pull request by Sillhouette**
I was not happy with the way I was handling the displaying of the board. It felt clunky and didn't seem to line up with concerns that properly. Due to the fact that displaying things is the job of the cli and providing the data for what should be placed on the board is the job of the board it seemed more appropriate to pass the data to a method in the cli class which proceeds to display the board appropriately. 

This decision was made due to a concern Ian brought up about the board's border variables not really belonging in the board class as well as the workshop I attended yesterday about simple design.

### CODE REVIEW

File: src/board.py [hunk 1]
```diff
@@ -1,25 +1,10 @@
 class Board:
     SIZE = 9
-    border = "⊱ –––––– {⋆⌘⋆} –––––– ⊰",
-    divider = "      ---+---+---"
     space = " "
```
Comment (dcmoore) [inline src/board.py]: Seems like a blank space is also a visual concern. Have you considered making null the representation of blank space at the board level, and blank space being the visual representation of a null space?

Comment (Sillhouette) [reply src/board.py]: That's a really interesting point... Let me think a little bit about how to tackle that

**Review by dcmoore — APPROVED** [review]

**Comment by dcmoore** [comment]
If you'll notice, you now have a merge conflict. I believe it's due to your whitespace.

**Comment by Sillhouette** [comment]
> If you'll notice, you now have a merge conflict. I believe it's due to your whitespace.

The merge request was due to the code in the other branch that I had just merged into master. Both of the branches worked on the same files in different ways

**Pull request merged**